### DISCUSSION TITLE
About config with keep_days=N

### DISCUSSION THREAD

**Discussion by John8974** [ROBelgium/MSNoise#323] [Q&A]
Hi, I've used MSNoise very well and appreciate that.
I felt very burdened with the hard-disk when I always save the outputs when using MSNoise.

But when I use 'keep_days=N' which means I will not save the results, I cannot plot mwcs, dtt results and also calculate dv/v based on dt/t. Maybe because plotting and calculation are based on the results in hard disk.

Is there any ways to get or see the results (stack, mwcs, dtt(for calculation of dv/v(necessary))) when using 'keep_days=N'?

**Comment by ThomasLecocq** [comment]
no, msnoise is built to work in steps, and between the steps, the hard disk is used.
